### PR TITLE
Asadchev/feature/einsum ta dot

### PR DESCRIPTION
**Pull request by asadchev**

### CODE REVIEW

**Comment by evaleev** [comment]
@asadchev seems to be WIP? (the einsum_dot  test is disabled)

**Comment by asadchev** [comment]
It's failing in parallel with scalapack enabled,  so i just gave up on it

File: tests/einsum.cpp [hunk 1]
```diff
@@ -723,4 +723,19 @@ BOOST_AUTO_TEST_CASE(einsum_tiledarray_replicated) {
   );
 }
 
+ BOOST_AUTO_TEST_CASE(einsum_tiledarray_dot) {
```
Comment (evaleev) [inline tests/einsum.cpp:726]: @asadchev I uncommented this ... should I comment it out?

**Comment by asadchev** [comment]
it fails parallel test that's why i commented it out

**Pull request merged**